packages/core/lib/src/data/css.dart

```diff
@@ -71,19 +71,19 @@ class CssBorder {
       );
 
   /// Calculates [Border].
-  Border getValue(TextStyleHtml tsh) {
+  Border? getValue(TextStyleHtml? tsh) {
```

Comment:
I think all the `tsh` params in `getValue` methods should be non-nullable.

```suggestion
  Border? getValue(TextStyleHtml tsh) {
```